schema/deploy/functions/hamming_distance.sql

```diff
@@ -39,10 +67,27 @@ create or replace function public.hamming_distance_ci(a text, b text)
     immutable
     parallel safe;
 
+create or replace function public.hamming_distance_ci_lte(a text, b text, threshold integer)
```

Comment:
Do we use the "ci" version of the function anywhere? If not it should be removed to follow your mantra of don't add it unless it's needed.

Reply:
Good point. We don't, AFAICT by searching across our repos and within Metabase. I'll remove both "ci" variants.

Reply:
Done in the repush. `hamming_distance_ci_lte` is never added and `hamming_distance_ci` is removed when it's lone callsite is removed.

Reply:
That's awesome! Thanks for doing that.